crates/noirc_evaluator/src/ssa/optimizations.rs

```diff
@@ -94,7 +96,49 @@ fn evaluate_intrinsic(
                     let i = Instruction::new(op, ObjectType::NotAnObject, Some(block_id));
                     result.push(ctx.add_instruction(i));
                 }
-                return result;
+                return Ok(result);
+            }
+            unreachable!();
+        }
+        builtin::Opcode::ToRadix(endian) => {
+            let mut element: Vec<u8>;
+            match args[0].to_biguint() {
+                Some(biguint) => {
+                    element = biguint.to_radix_le(args[1] as u32);
+                }
+                None => return Ok(Vec::new()),
+            }
+            let byte_count = args[2] as u32;
+            let diff = if byte_count > element.len() as u32 {
+                byte_count - element.len() as u32
+            } else {
+                return Err(RuntimeErrorKind::NotEnoughSpace {
```

Comment:
We need to write the 'bytes' to the output array, but the output does not have enough space. To me this is an Array-of-Bound error and we do not need to create a new error just for ToRadix.